python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -215,6 +217,7 @@ class GrapheneAssetNode(graphene.ObjectType):
         partitions=graphene.List(graphene.String),
     )
     materializedPartitions = graphene.NonNull(GrapheneMaterializedPartitions)
+    assetPartitions = graphene.NonNull(GrapheneAssetPartitions)
```

Comment:
this vs assetPartitionStatuses?

Reply:
prefer assetPartitionStatuses as this currently sounds like a list of the partition keys